cli/cli/helpers/engine_manager/engine_manager.go

```diff
@@ -247,6 +260,18 @@ func (manager *EngineManager) StartEngineIdempotentlyWithCustomVersion(ctx conte
 	}
 
 	clusterType := manager.clusterConfig.GetClusterType()
+
+	var lokiSink logs_aggregator.Sinks
+	var grafanaUrl string
+	if manager.clusterConfig.GetGraflokiConfig().ShouldStartBeforeEngine {
```

Comment:
Odd that you have to repeat yourself here and above. Shouldn't StartEngineManagerIdempotentlyWith(Current|Custom)Version share a codepath?

Reply:
agreed - could probably be refactored to use the same code path but going to leave this for a separate change